applications/luci-app-tor-hs/htdocs/luci-static/resources/view/tor_hs_common.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+'require view';
+'require ui';
+'require rpc';
+'require form';
+
+var callTorHSList = rpc.declare({
+			object: 'tor-hs-rpc',
```

Comment:
The indentation does not fit. only one tab and the parenthesis at the end of a new line. Please look at other modules how they do this